<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border: none;
	background-color: #transparent;
	background: none;
	padding: 0;
	margin: 0;
	color: #fff;
}

#centralwidget{
	background-color: #1f232a;
}

#tableWidget{
	background-color: #1f232a;
}

#leftMenuSubContainer{
	background-color: #16191d;
}

#leftMenuSubContainer QPushButton{
	text-align: left;
	padding: 2px 10px;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
}

#updateButton{
	background-color: #1f232a;
	border-top-right-radius: 10px;
}

#searchBarContainer QFrame{
	background-color: rgb(98, 98, 98);
	padding: 2px 2px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="leftMenuContainer" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QWidget" name="leftMenuSubContainer" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignTop">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="menuBtn">
               <property name="toolTip">
                <string>Menu</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons_light/icons_light/align-justify.svg</normaloff>:/icons_light/icons_light/align-justify.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignTop">
           <widget class="QFrame" name="frame_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QPushButton" name="homeBtn">
               <property name="toolTip">
                <string>Home</string>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #1f232a;</string>
               </property>
               <property name="text">
                <string>Home</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons_light/icons_light/home.svg</normaloff>:/icons_light/icons_light/home.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="dataBtn">
               <property name="toolTip">
                <string>Data Analysis</string>
               </property>
               <property name="text">
                <string>Data Analysis</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons_light/icons_light/list.svg</normaloff>:/icons_light/icons_light/list.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reportBtn">
               <property name="toolTip">
                <string>View Reports</string>
               </property>
               <property name="text">
                <string>Reports</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons_light/icons_light/printer.svg</normaloff>:/icons_light/icons_light/printer.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignBottom">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QPushButton" name="settingsBtn">
               <property name="toolTip">
                <string>Go to settings</string>
               </property>
               <property name="text">
                <string>Settings</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons_light/icons_light/settings.svg</normaloff>:/icons_light/icons_light/settings.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="infoBtn">
               <property name="toolTip">
                <string>Info about the app</string>
               </property>
               <property name="text">
                <string>Information</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons_light/icons_light/info.svg</normaloff>:/icons_light/icons_light/info.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="helpBtn">
               <property name="toolTip">
                <string>Get more help</string>
               </property>
               <property name="text">
                <string>Help</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons_light/icons_light/help-circle.svg</normaloff>:/icons_light/icons_light/help-circle.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="mainBodyContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="homePage">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="homeHeaderContainer" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: #1f232a;</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item alignment="Qt::AlignTop">
               <widget class="QWidget" name="homeHeaderLabelContainer" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QFrame" name="homeLabelFrame">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_9">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="homeLabel">
                      <property name="font">
                       <font>
                        <pointsize>18</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Interact E-Transfer List</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="homeUpdateFrame">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_8">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item alignment="Qt::AlignRight">
                     <widget class="QPushButton" name="updateButton">
                      <property name="font">
                       <font>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Update</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="homeSearchContainer" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QWidget" name="searchBarContainer" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QFrame" name="searchLineFrame">
                      <property name="styleSheet">
                       <string notr="true"/>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_11">
                       <property name="spacing">
                        <number>0</number>
                       </property>
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QLineEdit" name="lineEdit">
                         <property name="font">
                          <font>
                           <pointsize>16</pointsize>
                          </font>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">background-color: rgb(57, 57, 57);</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QFrame" name="searchButtonFrame">
                      <property name="styleSheet">
                       <string notr="true">border-top-right-radius: 10px;
border-bottom-right-radius: 10px;</string>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <property name="lineWidth">
                       <number>1</number>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_10">
                       <property name="spacing">
                        <number>0</number>
                       </property>
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>3</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QPushButton" name="pushButton">
                         <property name="font">
                          <font>
                           <pointsize>14</pointsize>
                          </font>
                         </property>
                         <property name="styleSheet">
                          <string notr="true"/>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                         <property name="icon">
                          <iconset resource="resources.qrc">
                           <normaloff>:/icons_light/icons_light/search.svg</normaloff>:/icons_light/icons_light/search.svg</iconset>
                         </property>
                         <property name="iconSize">
                          <size>
                           <width>24</width>
                           <height>24</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>30</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="homeBodyContainer" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QScrollArea" name="scrollArea">
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>900</width>
                   <height>519</height>
                  </rect>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QTableWidget" name="tableWidget">
                    <property name="autoFillBackground">
                     <bool>true</bool>
                    </property>
                    <column>
                     <property name="text">
                      <string>Name</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Amount</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Memo</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Date</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Ref_ID</string>
                     </property>
                    </column>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
          <zorder>homeBodyContainer</zorder>
          <zorder>homeHeaderContainer</zorder>
         </widget>
         <widget class="QWidget" name="dataPage"/>
         <widget class="QWidget" name="reportPage"/>
         <widget class="QWidget" name="settingPage"/>
         <widget class="QWidget" name="infoPage"/>
         <widget class="QWidget" name="helpPage"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
